__tests__/app.test.js

```diff
@@ -85,6 +85,62 @@ describe('/api/articles', () => {
               });
         });
 
+        test('PATCH:200 updates an article.votes by 1 given an article_id', () => {
+            const newVotes = {
+                inc_votes: 1
+            }
+            
+            return request(app)
+            .patch('/api/articles/3')
+            .send(newVotes)
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article.votes).toBe(1)
+            })
+        })
+
+        test('PATCH:200 updates an article.votes by 5 given an article_id', () => {
+            const newVotes = {
+                inc_votes: 5
+            }
+            
+            return request(app)
+            .patch('/api/articles/3')
+            .send(newVotes)
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article.votes).toBe(5)
+            })
+        })
+
+        test('PATCH:400 sends an appropriate status and error message when given an invalid article_id', () => {
+            const newVotes = {
+                inc_votes: 5
+            }
+
+            return request(app)
+              .patch('/api/articles/banana')
+              .send(newVotes)
+              .expect(400)
+              .then(({ body }) => {
+                expect(body.msg).toBe('Bad request');
+              });
+        });
+
+        test('PATCH:404 sends an appropriate status and error message when given a non-existent article_id', () => {
+            const newVotes = {
+                inc_votes: 5
+            }
+            
+            return request(app)
+              .patch('/api/articles/88999')
+              .send(newVotes)
+              .expect(404)
+              .then(({ body }) => {
+                expect(body.msg).toBe('Article does not exist');
+              });
+        });
```

Comment:
Missing a test for problems in the body

Status 400: Bad request, incomplete body
E.g. inc_votes value is not a number.